hologram_dashboard/extension/src/app/request-handler.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { DashboardService } from './dashboard.service'
+
+@Injectable()
+export class RequestHandler {
```

Comment:
I personally prefer DashboardService.

For server, it is handling the request. For the client, it is to call server to get the data.

Reply:
Here is the difference between components and services in https://angular.io/guide/architecture-services
A component can delegate certain tasks to services, such as fetching data from the server, validating user input, or logging directly to the console.